config/schedule.rb

```diff
@@ -0,0 +1,14 @@
+# Use this file to easily define all of your cron jobs.
+#
+# It's helpful, but not entirely necessary to understand cron before proceeding.
+# http://en.wikipedia.org/wiki/Cron
+
+# Example:
+#
+# set :output, "/path/to/my/cron_log.log"
+#
+
+# Learn more: http://github.com/javan/whenever
+every :day, at: '0:30 am' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.